src/pages/gen2/build-a-backend/data/customize-authz/public-data-access/index.mdx

```diff
@@ -40,3 +60,23 @@ const schema = a.schema({
     .authorization([a.allow.public('iam')]),
 });
 ```
+
+In your application, you can perform CRUD operations against the model using `client.models.<ModelName>` with the `iam` auth mode. Note in this instance the user must be unauthenticated for the client to sign the request using the Amplify-generated `UnAuthRole` IAM role.
```

Comment:
I think this is too technical. Things I'd call out here is to make sure the user has guest access enabled on the amplifyconfiguration.json. Something similar to this in the CDK tab https://docs.amplify.aws/react/build-a-backend/graphqlapi/customize-authorization-rules/#public-data-access but maybe put it into an accordion.
![image](https://github.com/aws-amplify/docs/assets/4989523/1c2ed94b-9749-4b1c-9769-c6497de206c2)